### PR TITLE
metrics: Add remote parallel bandwidth test

### LINKED ISSUE #712: Add remote networking parallel bandwidth test
Add remote networking parallel bandwidth test (-P4) using swarm along with iperf3.

### PR DESCRIPTION
**Pull request by GabyCT**
This will measure parallel bandwidth (-P4) with swarm and iperf3.

Fixes #712

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by chavafg** [comment]
lgtm

**Comment by chavafg** [comment]
@iphutch can you please take a look?

File: metrics/network/remote_network/remote-networking-iperf3.sh [hunk 1]
```diff
@@ -128,9 +151,12 @@ function main {
 		remote_network_bandwidth_iperf3
 	elif [ "$jitter_test" == "1" ]; then
 		remote_network_jitter_iperf3
+	elif [ "$parallel_test" == "1" ]; then
+		remote_network_parallel_iperf3
```
Comment (MarioCarrilloA) [inline metrics/network/remote_network/remote-networking-iperf3.sh]: why do you verify a var for calling a function and it is not called inside the case?

**Comment by MarioCarrilloA** [comment]
lgtm

**Pull request merged**